docs/tutorials/typescript/nextjs/index.md

```diff
@@ -1,290 +1,626 @@
+To connect to a Temporal Service using mTLS authentication, you will need to configure the connection address, Namespace, and mTLS certificates and keys.
+
+Review [Run Workers with Temporal Cloud for the TypeScript SDK](/getting_started/typescript/run_workers_with_cloud_typescript/index.md) for more details on creating certificates and connecting to Temporal Cloud.
+
+In this application, you'd change `temporal/src/worker.ts` to add certificate information:
 
 ```ts
-// before Worker.create call in worker.ts
 const connection = await NativeConnection.connect({
   address,
   tls: {
-    serverNameOverride,
-    serverRootCACertificate,
     clientCertPair: {
       crt: fs.readFileSync(clientCertPath),
       key: fs.readFileSync(clientKeyPath),
     },
   },
 });
+```
 
-// inside each Client call inside API Route
-const connection = await Connection.connect({
-  address,
-  tls: {
-    serverNameOverride,
-    serverRootCACertificate,
-    clientCertPair: {
-      crt: fs.readFileSync(clientCertPath),
-      key: fs.readFileSync(clientKeyPath),
+Then you'd update the `makeClient` function in `temporal/src/client.ts` to include the same keys:
+
+```ts
+function makeClient(): Client {
+  const connection = Connection.lazy({
+    address: 'localhost:7233',
+    tls: {
+      clientCertPair: {
+        crt: fs.readFileSync(clientCertPath),
+        key: fs.readFileSync(clientKeyPath),
+      },
     },
-  },
-});
+  });
+  return new Client({ connection });
+}
 ```
 
-[See the mTLS tutorial](https://docs.temporal.io/typescript/security#mtls-tutorial) for full details, or get in touch with us on Slack if you have reached this stage.
-
-## Production Concerns
+Restart your application to establish the connections to the new Temporal Service.
 
-As you move into production with your app, please review our docs on:
+## Conclusion
 
-- [Securing](https://docs.temporal.io/typescript/security)
-- [Testing](https://docs.temporal.io/typescript/testing)
-- [Patching](https://docs.temporal.io/typescript/patching) (aka migrating code to new versions)
-- [Logging](https://docs.temporal.io/typescript/logging)
-- [Production Deploy Checklist](https://docs.temporal.io/typescript/production-deploy)
+At this point, you have a working full stack example of a Temporal Workflow running inside your Next.js app, and the beginnings of an order processing system.
+From here you can add more Activities to the Workflow, or use this project as the basis for a different kind of application that needs long-running processes.
 
-You will also want to have a plan for **monitoring and scaling your Temporal Workers** that host and execute your Activity and Workflow code (separately from monitoring and scaling Temporal Server itself).
+You can also explore adding [Signals](https://docs.temporal.io/dev-guide/typescript/features//#signals) or  [Queries](https://docs.temporal.io/dev-guide/typescript/features//#queries) to your Workflow and map those to new API routes.
```

Comment:
We can quickly explain what signals and queries are in a sentence without encouraging click away from the page. How about:

"Temporal also allows you to add [Signals](https://docs.temporal.io/dev-guide/typescript/features//#signals) to your code to to send asynchronous notifications to running Workflows. You can also add [Queries](https://docs.temporal.io/dev-guide/typescript/features//#queries) to provide the user the ability to check the state of a Workflow at any point during its execution without affecting the Workflow's state. You can explore adding Signals and Queries to your Workflow and map those to new API routes."